tests/e2e/yamcs/telemetryTables.e2e.spec.mjs

```diff
@@ -53,4 +53,34 @@ test.describe("Telemetry Tables tests @yamcs", () => {
         await expect(page.getByRole('button', { name: 'SHOW LIMITED' })).toBeVisible();
     });
 
+    test.only('Telemetry tables when changing mode, will not change the sort order of the request', async ({ page }) => {
```

Comment:
Whoops :)

Reply:
Fixed. The funny thing is, I didn't even need it because I was using watch. sigh.